components/Testimonials.js

```diff
@@ -28,7 +28,7 @@ export default function Testimonials({ data }) {
                     />
                     <div>
                       <Link
-                        className="font-semibold text-gray-900 dark:text-gray-300"
+                        className="font-semibold text-primary-high dark:text-gray-300"
```

Comment:
I think this `dark:text-gray-300` should be one of our pre-made classes?

Reply:
I will merge and this can be fixed in the next PR

Reply:
Also I think the buttons on the homepage still have numbers or their colours and not the new classes